internal/pkg/mapper/sharedcode/codes/save.go

```diff
@@ -56,24 +56,25 @@ func (w *writer) save() error {
 		return err
 	}
 
-	// Content must be string or array
-	var codeContent string
-	if codeContentStr, ok := raw.(string); ok {
-		codeContent = strhelper.TransformationScriptsToString([]string{codeContentStr}, targetComponentId)
-	} else if codeContentSlice, ok := raw.([]interface{}); ok {
-		var scripts []string
-		for _, script := range codeContentSlice {
-			scripts = append(scripts, cast.ToString(script))
-		}
-		codeContent = strhelper.TransformationScriptsToString(scripts, targetComponentId)
-	} else {
-		return fmt.Errorf(`key "%s" must be string or string[], found %T, in %s`, model.ShareCodeContentKey, raw, w.configRow.Desc())
+	// Content must be []interface{}
+	codeContentSlice, ok := raw.([]interface{})
```

Comment:
Kedze sa to riesi uz pri `remote load`, a uz to bolo vyriesene pre `local load`, tak tu uz kontrola byt nemusi. ... ocakavame iba `[]interface{}`.